src/main/resources/templates/add-edit.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta charset="UTF-8">
+  <title>Amazin Bookstore</title>
+
+  <!-- Use Tailwind for CSS -->
+  <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+  <div>
```

Comment:
I think you can collapse both forms into one using conditionals in your `th:value` options.

ex: `<input type="text" name="isbn" class="whatever" th:value="${book == null ? '' : book.getIsbn()}" th:disabled="${book == null}">`

Reply:
I try to do so, but there are some problems with the disabled field for it. That field is conditional diable, but it's the opposite of the book == null condition. We want it to be not disabled when the book is null. But I don't know how to change that in the th:disabled. Also, thymeleaf don't recognize book != null for some odd reason.

Reply:
It doesn't work with `th:disabled`?

Reply:
from how i try it, it seems like so

Reply:
I've gotten it to work locally, are you sure you're putting the right condition? `th:disabled="${book != null}"` or try it with the inverse condition to see if you can edit the "locked" book fields

Reply:
I've gotten it to work locally, are you sure you're putting the right condition? `th:disabled="${book != null}"` or try it with the inverse condition to see if you can edit the "locked" book fields

Reply:
I've changed it and now it's working